Gopkg.toml

```diff
@@ -34,9 +34,9 @@
   version = "1.2.1"
 
 [[constraint]]
-  revision = "a73b8999d89d80bdcd89d199f41ab7fcdfd12dde"
+  revision = "bbee8bc8c4717ba17fc9186ff78ddb415a4370dc"
   name = "github.com/xsleonard/go-merkle"
-  source = "github.com/mikiquantum/go-merkle"
+  source = "github.com/touhonoob/go-merkle"
```

Comment:
Can you submit a PR from your fork to mikiquantum's fork? That's the fork we are currently using.

Reply:
PR on mikiquantum fork: https://github.com/mikiquantum/go-merkle/pull/1